### PR TITLE
[domd] Introduce AOS_VIS_PLUGINS variable to select required VIS plugins

### PR DESCRIPTION
**Pull request by al1img**
By default VIS is build with telemeteryemulatoradapter plugin. If other
plugin is required, it could be specified with AOS_VIS_PLUGINS
in local.conf

Signed-off-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

### CODE REVIEW

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend [hunk 1]
```diff
@@ -2,11 +2,23 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
...
-    file://visconfig.json \
+    file://visconfig_telemetryemulator.json \
+    file://visconfig_renesassimulator.json \
 "
+
+AOS_VIS_PLUGINS ?= "\
+    telemetryemulatoradapter \
+"
+
+PLUGINS = "\
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend]: ?=

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend [hunk 2]
```diff
@@ -17,7 +29,14 @@ FILES_${PN} += " \
 
 do_install_append() {
     install -d ${D}/var/aos/vis
-    install -m 0644 ${WORKDIR}/visconfig.json ${D}/var/aos/vis
+
+    if ${@bb.utils.contains('AOS_VIS_PLUGINS', 'telemetryemulatoradapter', 'true', 'false', d)}; then
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend]: quotes

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend [hunk 3]
```diff
@@ -17,7 +29,14 @@ FILES_${PN} += " \
...
+        install -m 0644 ${WORKDIR}/visconfig_telemetryemulator.json ${D}/var/aos/vis/visconfig.json
+    fi
+
+    if ${@bb.utils.contains('AOS_VIS_PLUGINS', 'renesassimulatoradapter', 'true', 'false', d)}; then
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend]: quotes

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend (hunk 1 above)
Comment (al1img) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend]: done

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend (hunk 2 above)
Comment (al1img) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend]: done

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend (hunk 3 above)
Comment (al1img) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-aos/aos-vis/aos-vis_git.bbappend]: done

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Pull request merged**